imagersite/imagersite/forms.py

```diff
@@ -0,0 +1,25 @@
+"""Docstring."""
+from django.forms import ModelForm
+from imager_profile.models import ImagerProfile
+# from django.conf import settings
+from django.contrib.auth.models import User
+
+
+class UserForm(ModelForm):
+    """Form to edit profile."""
+
+    class Meta:
+        """Docstring."""
+
+        model = User
+        fields = ["first_name", "last_name", "email"]
+
+
+class ProfileForm(ModelForm):
+    """Form to edit profile."""
+
+    class Meta:
+        """Docstring."""
+
+        model = ImagerProfile
+        exclude = ["user", "friends"]
```

Comment:
shouldn't these forms be part of the imager_profile app?  I would expect them to be there, given that the purpose of that app is managing the user profile.